src/Symfony/Component/Lock/Store/NullStore.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Lock\Store;
+
+use Symfony\Component\Lock\BlockingSharedLockStoreInterface;
+use Symfony\Component\Lock\Key;
+
+/**
+ * NullStore is a PersistingStoreInterface implementation which discards all operations.
+ *
+ * @author Pavel Bartoň <barton@webwings.cz>
+ */
+class NullStore implements BlockingSharedLockStoreInterface
```

Comment:
Introducing this class (as well as the changes to the `StoreFactory`) must be moved into a separate PR targeting the `7.2` branch. That's a new feature.